### ISSUE TITLE
Adding Right Click Context Menu Items

### ISSUE THREAD

**Issue by yusufani** [aleksey-hoffman/sigma-file-manager#14]
## Feature request:
First of all, the application looks very nice, congratulations on that.

There are very useful shortcuts in the context menu that we open with a right click in Windows file explorer. A section can be great for moving or adding these shortcuts.

What I am talking about : 
![image](https://user-images.githubusercontent.com/35346311/119516239-44efe200-bd7f-11eb-83c6-7cce664f88b6.png)

**Comment by thebluedev** [comment]
It seems that the application specific context menu items are missing 
![image](https://user-images.githubusercontent.com/73629408/119522669-cbb1b880-bd99-11eb-8b37-99e2906de250.png)

the normal context menu when not clicking on a file but directly onto the directory is also not present
![image](https://user-images.githubusercontent.com/73629408/119522290-77a6d400-bd99-11eb-923d-01e35c6a6a02.png)
creating a new file or folder is meticulous, as you have to go all the way to the top left and select new file/folder, would be better if it was just added to the right-click menu

**Comment by radholm** [comment]
Yes, this is a must have feature IMO. Me myself need the context menu in order to quickly compile AHK scripts.
In terms of Windows, context menu references are found in the registry, so this would involve platform specificity.

**Comment by aleksey-hoffman** [comment]
Hey Yusuf, I will add this later (possibly in v1.6).
BTW, you can already add custom programs to the app list in the context menu

![image](https://user-images.githubusercontent.com/61761672/183263532-d7f359cc-11fc-4c8c-969e-c720adea877b.png)
